mapreduce/src/main/java/org/pentaho/hadoop/mapreduce/PentahoMapRunnable.java

```diff
@@ -31,9 +31,7 @@
 import org.apache.hadoop.mapred.Reporter;
 import org.pentaho.di.core.Const;
 import org.pentaho.di.core.exception.KettleException;
-import org.pentaho.di.core.logging.KettleLogStore;
-import org.pentaho.di.core.logging.KettleLoggingEvent;
-import org.pentaho.di.core.logging.LogLevel;
+import org.pentaho.di.core.logging.*;
```

Comment:
Fix this import; only import specific packages.

Reply:
Yes. I will. It is what IntelliJ did automatically